gitsign.yaml

```
@@ -13,6 +13,18 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: "${{package.name}}-config"
+    description: "GitSign config"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/etc/
+          git config --global commit.gpgsign true       # Sign all commits
+          git config --global tag.gpgsign true          # Sign all tags
+          git config --global gpg.x509.program gitsign  # Use gitsign for signing
+          git config --global gpg.format x509           # gitsign expects x509 args
```

```suggestion
          git config --system commit.gpgsign true       # Sign all commits
          git config --system tag.gpgsign true          # Sign all tags
          git config --system gpg.x509.program gitsign  # Use gitsign for signing
          git config --system gpg.format x509           # gitsign expects x509 args
```

I think we probably want `--system` here so it configures for any user. (`--global` is global for the current user)

---

done